### DISCUSSION TITLE
People detected in motion masked area.

### DISCUSSION THREAD

**Discussion by rspierenburg** [blakeblackshear/frigate#10612] [Ask A Question]
### What is your question:

I was under the impression that if Frigate detects motion it would then call on the detector to determine if the object that moved matches one of the listed options in which case it would then create an event.
I have setup a front porch cam and motion masked off the street, neighbors driveway etc.  However I still seem to be getting "Persons" event notifications when the are clearly in the masked off region.
Should I copy the motion mask I have to the person mask as well?

**Comment by NickM-27** [comment]
Please read the docs https://docs.frigate.video/configuration/masks this is not how motion masks are meant to be used.

If you only want events created when an object enters an area of interest then you should be using required zones to tell frigate about the areas that you do care about https://docs.frigate.video/configuration/zones

**Reply by rspierenburg** [reply]
Ah gotcha, yes I was going about this all wrong.  Thanks for helping me clear this up.
